src/tests/JIT/opt/Regressions/Regression1.cs

```diff
@@ -0,0 +1,72 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.CompilerServices;
+
+public class C0
+{
+   public short F0;
+   public bool F1;
+   public sbyte F2;
+   public short F3;
+   public C0(sbyte f2, short f3)
+   {
+       F2 = f2;
+       F3 = f3;
+   }
+}
+
+public class Program
+{
+   public static IRuntime s_rt;
+   public static int[][] s_13 = new int[][] { new int[] { 0 } };
+
+   public static int Main()
+   {
+       s_rt = new Runtime();
+       var result = M74(0);
+
+       if (result != -1)
+           return 0;
+
+       return 100;
+   }
+
+   [MethodImpl(MethodImplOptions.NoInlining)]
+   public static short M74(short arg1)
+   {
+       try
+       {
+           M75();
+       }
+       finally
+       {
+           C0 var5 = new C0(0, 1);
+           int var6 = s_13[0][0];
+           arg1 = var5.F3;
+           s_rt.WriteLine(var5.F0);
+           s_rt.WriteLine(var5.F1);
+           s_rt.WriteLine(var5.F2);
+       }
+
+       arg1 = (short)~arg1;
+       arg1++;
+       return arg1;
+   }
+
+   public static sbyte[] M75()
+   {
+       return default(sbyte[]);
+   }
+}
+
+public interface IRuntime
+{
+   void WriteLine<T>(T value);
+}
+
+public class Runtime : IRuntime
+{
+   public void WriteLine<T>(T value) { }
+}
```

Comment:
I don't really follow what this test is doing.  There's a bit of short arithmetic - do the try, M75 call, WriteLines, etc., all contribute to some codegen impact?

I'll have similar questions for the other tests like this, so I'll wait to hear what this is before trying to understand them.

Reply:
I see your comment above about the test being brittle (failing to repro with changes).  It seems like it is likely that a future JIT change will impact the IR so that this test won't be useful anymore.  I'm not sure what to do about that, though if there's nothing productive to do, perhaps having the conditions in the JIT that led to incorrect behavior listed in the test would be helpful?

Reply:
This test was generated by Fuzzlyn that caught an assertion/wrong value in the early implementation of this change. So I kept this test to make sure that it doesn't happen again. It's not checking for any disassembly, it is just to make sure it gives the correct result.